canonicalwebteam/discourse/models.py

```diff
@@ -147,6 +147,31 @@ def get_last_activity_time(self, topic_id):
 
         return result["rows"]
 
+    def get_categories_last_activity_time(self, category_id):
+        """
+        Uses data-explorer to the last time a specifc topic was updated
```

Comment:
Typo, but also should there be a verb here? Something like this?
```suggestion
        Uses data-explorer to get the last time a specific topic was updated
```